### PR TITLE
Adding quadratic-regression-analysis.md file

### LINKED ISSUE #4824: [Concept Entry] Sklearn quadratic-regression-analysis
### Reason/inspiration (optional)

We would like a new entry on the `Quadratic Regression Analysis` concept under Sklearn. The entry should go in a new file under `docs/content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md`.

The entry should include:

- An introduction to the concept
- A ‘Syntax’ section that provides the syntax for the concept (if applicable)
- An ‘Example’ section that provides an example demonstrating the concept in use
- A ‘Codebyte Example’ section that provides a codebyte example demonstrating the concept in use

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by uhck**
### Description

Adding quadratic-regression-analysis.md file

### Issue Solved

Closes #4824 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by uhck** [comment]
Could you check why it is failing at
`prettier "./**/*.(ts|md)" --list-different` for the file name
`content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md`
This is the requested new file name for this entry.

File: content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+poly = PolynomialFeatures(degree=2, interaction_only=True, include_bias=False)
+model = LinearRegression()
+
+# Fit the quadratic equation into the linear regression model with the training data and labels
+poly_features = poly.fit_transform(training_data)
+model.fit(poly_features, labels)
+
+# Predict labels for test data
+predictions = model.predict(test_data)
+```
```
Comment (mamtawardhani) [inline content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md:42]: This can be converted into an example rather than a syntax. could you please convert this to an example?

File: content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+PolynomialFeatures has parameters for:
+
+- `degree`: In this case, for a quadratic equation, it is 2 by default.
+- `interaction_only`: If True, only interaction features are produced; this includes only features that are products of single degree input features, terms with power of 2 or more are not included.
+- `include bias`: If True, the function will include a bias column in which all polynomial powers are 0 (the value of `c` in the quadratic equation above). We turn this off since we are plugging the poly features into a linear regression model which will take this into consideration by default.
```
Comment (mamtawardhani) [inline content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md:48]:
```suggestion
- `degree`: Specifies the degree of polynomial features to generate, determining how high the polynomial terms should go (e.g., 2 for quadratic).
- `interaction_only`: When set to `True`, generates only interaction features, excluding individual polynomial terms.
- `include_bias`: If `True`, includes a bias column with all ones to represent the constant term; otherwise, the bias term is omitted.
```

File: content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+LinearRegression has parameters for:
+
+- `fit_intercept`: If True, the function will calculate the intercept for the model.
+- `copy_X`: If True, the function will create a deep copy of X, otherwise it may be overwritten.
+- `n_jobs`: Specifies the number of parallel jobs to compute the model if more than 1 is needed for larger problems.
+- `positive`: If True, forces the coefficients to be positive. This may help identify a more correct model in the presence of noisy data but introduces potential bias.
```
Comment (mamtawardhani) [inline content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md:55]:
```suggestion
- `fit_intercept`: Determines whether the model should calculate an intercept term; if `False`, the model is forced through the origin.
- `copy_X`: If `True`, creates a deep copy of the input data to avoid modifying the original; otherwise, the input data might be overwritten.
- `n_jobs`: Specifies the number of CPU cores to use for parallel computations; -1 uses all available cores.
- `positive`: If `True`, ensures that all coefficients are constrained to be positive values.
```

File: content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+## Codebyte Example
+
```
Comment (mamtawardhani) [inline content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md:58]:
```suggestion

Run the following codebyte to understand quadratic regression by fitting a linear model to polynomial features:

```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @uhck thank you for contributing to Codecademy Docs, the entry looks well written 😃 

I've suggested a few modifications, could you please review and modify them at your earliest convenience? Thank you! 😄

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @uhck thank you for making the changes. I am requesting one last change in the image. Could you correct the image such that the right hand side is not Zero but `y`. This is the equation:
`y=ax^2 + bx + c`

**Comment by uhck** [comment]
Updated the image

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing @uhck! 😄 

The entry looks good for a second round of review 🚀

**Review by Sriparno08 — APPROVED** [review]
Looks good, @uhck! 👍
